bpf/init.sh

```diff
@@ -34,6 +34,9 @@ set -x
 # Enable JIT
 echo 1 > /proc/sys/net/core/bpf_jit_enable
 
+# Disable rp_filter
+echo 0 > /proc/sys/net/ipv4/conf/all/rp_filter
```

Comment:
Yes, you are right, it's the max value of "all" and the interface that is used. I'm reluctant to change this setting for "all" although we keep "default" as-is as it might have implications outside of the interfaces we create & manage.

I think at the very least we should print a big fat warning. I'll see if we can change the BPF programs to change the pkt_type of packets so they are always considered local and skip the rp_filter check.